.github/workflows/ci.yml

```
@@ -322,28 +334,50 @@
         # npm install -g wait-on
         # npx wait-on http://localhost:5500 -t 60000 -l
         # curl http://localhost:5500 -vd
 
-        
+  # TEST SCALA CODING
+  test-scala-coding:
+    needs: build-scala-latest
+    runs-on: ubuntu-latest
+    environment: CI
+    steps:
+    - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)